### PR TITLE
OTT-1889: Agent table-level statistics implementation

### PR DESCRIPTION
**Pull request by YuzeLiao**

### CODE REVIEW

File: driver/collector/base_collector.py [hunk 1]
```diff
@@ -27,6 +27,10 @@ def collect_metrics(self) -> Dict[str, Any]:
     @abstractmethod
     def collect_table_row_number_stats(self) -> Dict[str, Any]:
         """Collect statistics about the number of rows of different tables"""
+    
+    @abstractmethod
+    def collect_table_level_metrics(self, num_table_to_collect_stas: int) -> Dict[str, Any]:
```
Comment (bohanjason) [inline driver/collector/base_collector.py:32]:
```suggestion
    def collect_table_level_metrics(self, num_table_to_collect_stats: int) -> Dict[str, Any]:
```

File: driver/collector/postgres_collector.py [hunk 2]
```diff
@@ -298,6 +298,23 @@ def collect_table_row_number_stats(self) -> Dict[str, Any]:
         raw_stats = self._cmd(self.ROW_NUMS_SQL)
         return {entry[0]: entry[1] for entry in zip(raw_stats[1], raw_stats[0][0])}
 
+    def collect_table_level_metrics(self, num_table_to_collect_stas: int) -> Dict[str, Any]:
+        """Collect table level statistics"""
+        return {
+            "data": {
+                "pg_stat_user_tables": {
+                    "columns": [
+                        "to_do",
```
Comment (bohanjason) [inline driver/collector/postgres_collector.py:307]:
```suggestion
                        "a", "b", "c",
```

File: driver/collector/mysql_collector.py [hunk 3]
```diff
@@ -229,6 +229,23 @@ def collect_table_row_number_stats(self) -> Dict[str, Any]:
         """Collect statistics about the number of rows of different tables"""
         return {}
 
+    def collect_table_level_metrics(self, num_table_to_collect_stas: int) -> Dict[str, Any]:
+        """Collect table level statistics"""
+        return {
+            "data": {
+                "mysql_to_implement": {
+                    "columns": [
+                        "mysql_to_implement",
```
Comment (bohanjason) [inline driver/collector/mysql_collector.py:238]:
```suggestion
                        "a", "b", "c",
```

**Review by bohanjason — CHANGES_REQUESTED** [review]
I think we should implement it in driver first.

File: driver/compute_server_client.py [hunk 4]
```diff
@@ -74,3 +89,33 @@ def post_observation(self, data: Observation) -> None:
...
+                # TODO: Remove this when compute-service has the endpoint
+
+        for key in data["data"].keys():
+            data["data"][key]["rows"] = data["data"][key]["rows"][:10]
+        logging.info(
+            "Ready to send data to compute-service. Data sample=%s",
+            data,
+        )
+        # pylint: disable=pointless-string-statement
+        """url = f"{self._server_url}/observation/"
```
Comment (ruogudu) [inline driver/compute_server_client.py:112]: Is this still a draft? Also the URL should be difference.

**Review by ruogudu — APPROVED** [review]

**Review by bohanjason — CHANGES_REQUESTED** [review]

**Review by bohanjason — APPROVED** [review]

**Pull request merged**